.github/workflows/unit-test.yml

```diff
@@ -41,7 +41,8 @@ jobs:
       - name: Install Prerequisites
         run: |
           source $VENV
-          zenml integration install kubeflow s3 gcp azure gcp_secrets_manager vertex vault -f
+          zenml integration install kubeflow s3 gcp azure gcp_secrets_manager
+          vertex vault pillow -f
```

Comment:
does this still work when it's split over two lines like this?

Reply:
Hmmm good point. I've updated. I'm pretty sure it was like that before I got there, but in any case it's all on a single line now.